src/api/spec/controllers/staging/staging_projects_controller_spec.rb

```diff
@@ -139,4 +139,51 @@
                                                                                                            user.id)
     end
   end
+
+  describe 'POST #create' do
+    let(:staging_workflow_project) { staging_workflow.project.name }
+    let!(:other_project) { create(:project, name: "#{project}:other_project") }
+
+    before do
+      login(user)
+      staging_workflow
+    end
+
+    subject { post :create, params: { staging_workflow_project: staging_workflow_project, format: :xml }, body: body }
+
+    context 'succeeds' do
+      let(:body) do
+        <<~XML
+          <workflow>
+            <staging_project>#{project}:Staging:C</staging_project>
+            <staging_project>#{project}:other_project</staging_project>
+          </workflow>
+        XML
+      end
+
+      it { expect(subject).to have_http_status(:success) }
+      it { expect { subject }.to change(Project, :count).by(1) }
```

Comment:
IMO posting 2 names to create should either create 2 or error. Because if posting existing staging projects is fine, I would expect missing stagings to be removed (or at least unlinked - how do I unlink a project btw?).

Reply:
@coolo, at the moment, is not possible to unlink a project from a staging, but you are right we should provide a way to unlink